A72-open-telemetry-tracing.md

```diff
@@ -205,57 +186,75 @@ On attempt span:
   boolean value indicating whether the stream is undergoing a transparent retry.
 * If the RPC experienced load balancer pick delay, add an Event with the name 
   "Delayed LB pick complete" upon creation of the stream on the transport. 
-* When the application sends an outbound message to the transport, add an Event 
+* When the application sends an outbound message to the transport, add Event(s)
+  (it depends on implementation whether there is a single event or two separate 
+  events for compressed/uncompressed message sizes, the same below) 
   with name "Outbound message sent" and the following attributes:
```

Comment:
If the implementation chooses to record separate events for compressed and uncompressed message sizes, presumably the events will have different names.  We should probably specify the name of the other event (e.g., maybe something like "Outbound message compressed").

Also, can we specify anything about the order of the events if there are two of them?  Alternatively, if we think this will be different in different impls, we can say that the events can be in either order.